src/test/java/lotto/domain/model/LottosTest.java

```diff
@@ -16,10 +16,20 @@ class LottosTest {
     @Test
     void 당첨번호를_입력받으면_당첨번호와_몇개_당첨인지_알린다() {
         Lottos lottos = new Lottos(List.of(
-            new Lotto(List.of(1, 2, 3, 4, 5, 6)), new Lotto(List.of(40, 41, 42, 43, 44, 45))));
+            new Lotto(1, 2, 3, 4, 5, 6), new Lotto(40, 41, 42, 43, 44, 45)));
         
-        WinningResult winningResult = lottos.findWinners(new Lotto(List.of(35, 36, 37, 43, 44, 45)));
-        WinningResult exptedwinner = new WinningResult(1, 0, 0, 0);
-        assertThat(winningResult).isEqualTo(exptedwinner);
+        WinningResult winningResult = lottos.identifyWinners(new Lotto(35, 36, 37, 43, 44, 45), new Bonus(1));
+        WinningResult expectedWinner = new WinningResult(1, 1, 0, 0, 0, 0);
```

Comment:
```suggestion
        WinningResult expectedWinner = new WinningResult();
        expectedWinner.increaseRankCount(Rank.FIRST);
        expectedWinner.increaseRankCount(Rank.SECOND);
```

생성자를 통해 등수를 전달하니 몇 등인지를 파악하는데 어려움이 있을 것 같다.
위와 같이 구현하는 것은 어떨까?